easybuild/easyblocks/o/openssl_wrapper.py

```diff
@@ -368,10 +369,20 @@ def sanity_check_step(self):
             'dirs': ssl_dirs,
         }
 
+        # use proxy to connect if https_proxy environment variable is defined
+        proxy_arg = ''
+        if os.environ.get('https_proxy'):
+            # only use host & port from https_proxy env var, that is, strip
+            # any protocol prefix and trailing slashes
+            proxy_parsed = urlparse(os.environ.get('https_proxy'))
+            if proxy_parsed.netloc:
+                proxy_arg = f' -proxy {proxy_parsed.netloc}'
```

Comment:
EasyBuild still supports running with Python 2.7 and 3.5, which don't have f-string support

Reply:
@trz42 ping, please fix